Philips.CodeAnalysis.Test/Helpers/AssertCodeHelper.cs

```diff
@@ -45,10 +46,10 @@ public string GetText(string methodBody, string otherClass, string attributes)
 			return string.Format(TestTemplate, otherClass, attributes, methodBody);
 		}
 
-		public MetadataReference[] GetMetaDataReferences()
+		public ImmutableArray<MetadataReference> GetMetaDataReferences()
 		{
 			//add symbols for assert
-			return new[] { MetadataReference.CreateFromFile(typeof(Assert).Assembly.Location), MetadataReference.CreateFromFile(typeof(TimeoutAttribute).Assembly.Location) };
```

Comment:
I think this override is legacy and can be deleted. I think the MsTest classes are always available to the Unit Tests.

Reply:
Indeed, removed